### ISSUE TITLE
Different cells height with different row types

### ISSUE THREAD

**Issue by allaire** [xmartlabs/Eureka#620]
Hi,

It seems like the height of the cells are different according to their cell type, see screenshot : 

![img_1150](https://cloud.githubusercontent.com/assets/304461/17823770/44f4bf00-662c-11e6-91d3-f2ab26e2d047.jpg)

First two cells are `TextRow`, then `CheckRow` and finally `PushRow` (disabled).

`CheckRow` and `PushRow` does not have the same height as my `TextRow` cells. They have the same font, etc. which was modified with `defaultCellUpdate` for each row type.

Something I'm missing or Autolayout bug?

**Comment by allaire** [comment]
Same thing when I comment out all the style I do in `defaultCellUpdate` for all my cells, although less apparent:
 
![img_1151](https://cloud.githubusercontent.com/assets/304461/17824052/b7da2cf2-662d-11e6-87a7-ba978083d925.jpg)

Is there a way to set a minimum cell height?

**Comment by plastus** [comment]
func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
    return 40
}

I haven't tested this, but would this fix the issue?

**Comment by allaire** [comment]
@plastus AFAIK this is not compatible with expandable cells like `TimeInlineRow`

**Comment by plastus** [comment]
@allaire I've noticed this issue as well (pushRow cells being taller than all others) but haven't dug into the issue too much yet. If I run across a fix that doesn't break other functionality, I'll post an update here.

**Comment by allaire** [comment]
@plastus This is caused by the autolayout dynamic height, based on the cell's content. One way to fix this would be to add a `>= height constraint`, which could be optional of course. That way, we could define a min height for our cells, so it all looks the same :)

**Comment by mats-claassen** [comment]
Apparently it happens that those rows that define their Autolayout constraints have a different height than those that use the default `textLabel` and `detailTextLabel`. At least when setting a bigger font it gets more apparent that the former grow/resize differently from the latter. 

Setting `height` of each to to a value works and leaves all rows with the same height. This could be done in the `defaultCellSetup` of each row type if the same cell size is used accross the whole app.

I am not sure adding a `>= height constraint ` will work because we would have to add it to all the cells, right? And those which currently have no constraints but rely on frame positioning would have to be changed to rely completely on constraints as Autolayout constraints do not mix well with frame positioning.

For now, setting a correct `height` for every cell works although it could be tedious.

**Comment by allaire** [comment]
@mats-claassen Maybe we could add a bit more default padding top/bottom so it matches the default frame positioning cells?

**Comment by mats-claassen** [comment]
Hmm, but I think this padding would be different for each font size. With the default font size the height difference is less than 2 points but when the font size increases then it becomes much more.

**Comment by allaire** [comment]
@mats-claassen True. Is this the so-called "relative to margin" AutoLayout rule? I'm pretty sure there must be a way to create our own AL constraints that behave the same. Unfortunately, my google searched yielded no results ;(

**Comment by mats-claassen** [comment]
I don't think so. I believe the margins may change for the different devices but not for fonts.
